pkg/helpers/helpers.go

```diff
@@ -63,6 +63,12 @@ func PointerToBool(b bool) *bool {
 	return &p
 }
 
+// PointerToInt returns a pointer to a int
+func PointerToInt(i int) *int {
```

Comment:
This would be an easy unit test to add in https://github.com/Azure/acs-engine/blob/master/pkg/helpers/helpers_test.go

Reply:
added